optashift-employee-rostering-webapp/src/test/resources/arquillian.xml

```diff
@@ -0,0 +1,15 @@
+<?xml version="1.0" encoding="UTF-8" standalone="yes"?>
+<arquillian xmlns="http://jboss.org/schema/arquillian"
+            xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+            xsi:schemaLocation="http://jboss.org/schema/arquillian http://jboss.org/schema/arquillian/arquillian_1_0.xsd">
+
+  <container qualifier="wildfly-managed" default="true">
+    <configuration>
+      <property name="jbossHome">../local/appserver/wildfly-10.1.0.Final</property>
```

Comment:
Can we use `<property name="jbossHome">target/wildfly-${version.org.wildfly}</property>` just like in optaplanner-webexamples to avoid hard coding the wildfly version?

Reply:
No; the arquillian configuration file does not share properties with pom.xml. I got the error "jbossHome &apos;../local/appserver/wildfly-${version.org.wildfly}&apos; must exist" when using that setup.